apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -47,6 +47,7 @@ function render_fallback_coming_soon_page() {
 	remove_action( 'wp_head', 'global_css', 5 );
 	remove_action( 'wp_footer', 'wpcom_subs_js' );
 	remove_action( 'wp_footer', 'stats_footer', 101 );
+	remove_action( 'init', 'the_neverending_home_page_init', 20 );
```

Comment:
Unfortunately this didn't end up working on my test site, I think it digs its claws a bit deeper. I'll do a bit more digging to see if there are more specific actions we need to remove.

Reply:
Apologies @ramonjd, looks like the following should work on simple sites:

```
add_filter( 'infinite_scroll_archive_supported', '__return_false', 99 );` // Disable infinite scroll feature
```

Reply:
I blindly copy and paste and defer testing and you apologize? 🇨🇦 

Ha. Thanks again 🙇